docs/internal-architecture.md

```diff
@@ -0,0 +1,91 @@
+---
+description: Internal architecture of the project.
+---
+
+# Internal architecture
+
+### Trackers
+
+A tracker is usually in the type of an index tracker for a coin or token on a blockchain network.
```

Comment:
A tracker is actually just a process that gets data from an API - HTTP or Blockchain.